### PR TITLE
Feature/propertiesroutes

### PR DESCRIPTION
**Pull request by LennonSimpson**

### CODE REVIEW

File: controllers/properties-routes.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// Getting individual properties
```
Comment (DuGBB) [inline controllers/properties-routes.js:1]: @LennonSimpson why is this file needed?  Why wouldn't we use the get one function in `controllers/api/property-routes`?

Comment (LennonSimpson) [reply controllers/properties-routes.js:1]: I assumed this was for the front end, re: pages that show individual property details.

**Review by DuGBB — APPROVED** [review]
